<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>map</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>742</width>
      <height>392</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>30</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="text">
        <string>Floor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2">
       <property name="text">
        <string>Room</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton3">
       <property name="text">
        <string>Area</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton4">
       <property name="text">
        <string>Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton5">
       <property name="text">
        <string>Array</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>216</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>123</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton0">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
